crates/bevy_dev_tools/src/debug_overlay/inset.rs

```diff
@@ -0,0 +1,197 @@
+use bevy_color::Color;
+use bevy_gizmos::{config::GizmoConfigGroup, prelude::Gizmos};
+use bevy_math::{Vec2, Vec2Swizzles};
+use bevy_reflect::Reflect;
+use bevy_transform::prelude::GlobalTransform;
+use bevy_utils::HashMap;
+
+use super::{CameraQuery, LayoutRect};
+
+trait ApproxF32 {
+    fn is(self, other: f32) -> bool;
+}
+impl ApproxF32 for f32 {
+    fn is(self, other: f32) -> bool {
+        let diff = (self - other).abs();
+        diff < 0.001
```

Comment:
Extracting this into an EPSILON associated constant would be good.

Or TBH we could just refactor this as a free `approx_eq(f32, f32) -> bool` function and I think this would be simpler and reduce the temptation for reuse.

Reply:
I think I'll just remove this trait if I can, a simple function is probably better here.